torch/_thnn/utils.py

```diff
@@ -2,8 +2,8 @@
 import itertools
 import importlib
 
-THNN_H_PATH = os.path.join(os.path.dirname(__file__), '..', 'lib', 'THNN.h')
```

Comment:
these changes will come in via https://github.com/pytorch/pytorch/pull/8214, so you can remove them from this PR as well.